app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java

```diff
@@ -68,6 +69,15 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
         View root = inflater.inflate(R.layout.fragment_merge_files, container, false);
         ButterKnife.bind(this, root);
+        root.setFocusableInTouchMode(true);
+        root.requestFocus();
+        root.setOnKeyListener((v, keyCode, event) -> {
```

Comment:
There is no need to add this code in every fragment. You can just add `addToBackStack(null).` here 
https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/activity/MainActivity.java#L196

Make it this & it'll work fine
`fragmentManager.beginTransaction().addToBackStack(null).replace(R.id.content, fragment).commit();`